documentation/src/main/asciidoc/topics/ref_quick_start.adoc

```diff
@@ -8,7 +8,7 @@ management tooling, and either the {brandname} server or core libraries.
 
 To run {brandname}, you'll need:
 
-* Java 11 JDK at a minimum. {brandname} but also works with JDK 17, 18 and 19.
+* Java 11 JDK at a minimum. {brandname} also works with 17, 18, 19.
```

Comment:
```suggestion
* Java 11 JDK at a minimum.
```
AFAIK, we don't test with JDK 18/19. Speaking of testing, we may need a profile to test with JDK 11, right? 
cc @tristantarrant wdyt? is it worth adding such complexity?